backend/app/assets/stylesheets/spree/backend/components/_filters.scss

```diff
@@ -1,4 +1,5 @@
 #table-filter {
+  display: none;
```

Comment:
Properties should be ordered border-top-left-radius, border-top-right-radius, display, margin-top